src/harness/testcases/WINNF_FT_S_FPR_testcase.py

```diff
@@ -28,6 +30,483 @@ def setUp(self):
 
   def tearDown(self):
     pass
+
+  def generate_FPR_2_default_config(self, filename):
+    """Generates the WinnForum configuration for FPR.2."""
+
+    # Load FSS station operating at range below 3700 to 4200 MHz, TT&C flag ON.
+    fss_data = json.load(
+        open(os.path.join('testcases', 'testdata', 'fss_record_0.json')))
+    fss_record = fss_data['record']
+    fss_record['deploymentParam'][0]['operationParam'][
+        'operationFrequencyRange']['lowFrequency'] = 3625000000
+    fss_record['deploymentParam'][0]['operationParam'][
+        'operationFrequencyRange']['highFrequency'] = 4200000000
+    fss_data['ttc'] = True
+
+    # Load devices and grant info for Scenario 1.
+    # Loading device_1 (Cat A) to a location within 40 km of FSS,
+    # frequency range outside of FSS band.
+    device_1 = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_a.json')))
+    device_1['installationParam'][
+        'latitude'] = fss_record['installationParam']['latitude'] + 0.20
+    device_1['installationParam'][
```

Comment:
I'm getting a failure when trying to generate the configs.  I think you need to add 

['deploymentParam'][0]

when you are modifying the lat/longs of the cbsd based on the lat/longs of the fss..

device_1['installationParam'][
+        'latitude'] = fss_record['installationParam']['latitude'] + 0.20

between fss_record and ['installationParam']

Reply:
Oups, sorry about that. Corrected.